applications/CompressiblePotentialFlowApplication/python_scripts/compute_forces_on_nodes_process.py

```diff
@@ -29,7 +29,7 @@ def ExecuteFinalizeSolutionStep(self):
     def Execute(self):
         KratosMultiphysics.Logger.PrintInfo('ComputeForcesOnNodesProcess', 'Computing reactions on nodes')
 
-        KratosMultiphysics.VariableUtils().SetHistoricalVariableToZero(KratosMultiphysics.REACTION, self.body_model_part.Nodes)
+        KratosMultiphysics.VariableUtils().SetNonHistoricalVariableToZero(KratosMultiphysics.REACTION, self.body_model_part.Nodes)
```

Comment:
have you tested this? seemingly, set to zero does not work as long as the non historical var is not initialized, replacing it with 
" KratosMultiphysics.VariableUtils().SetNonHistoricalVariable(KratosMultiphysics.REACTION, KratosMultiphysics.Vector(3), self.body_model_part.Nodes)"
works for me

Reply:
that said, I am sure you tested but maybe we are missing to initialise correctly in our examples, can you give us a hint?

Reply:
Hi! This is working in my machine (tests are passing), and I get no problems with this function.  When you say that Set to Zero does not work, which error are you getting?

Regards!

Reply:
sorry, recompiling fixed the issue